### PR TITLE
tmuxPlugins.rose-pine: init at unstable-2024-01-08

### PR DESCRIPTION
**Pull request by axeltlarsson**
## Description of changes

Add tmux theme Rosé Pine.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by AxelTLarsson** [comment]
@ambroisie @Janik-Haag any chance of getting some eyes on this? 🙏

File: pkgs/misc/tmux-plugins/default.nix [hunk 1]
```diff
@@ -534,6 +534,24 @@ in rec {
...
+    pluginName = "rose-pine";
+    version = "unstable-2024-01-08";
+    rtpFilePath = "rose-pine.tmux";
+    src = fetchFromGitHub {
+      owner = "rose-pine";
+      repo = "tmux";
+      rev = "dd6d01338ac4afeb96542dcf24e4a7fe179b69e6";
+      sha256 = "sha256-Tccb4VjdotOSw7flJV4N0H4557NxRhXiCecZBPU9ICQ=";
+    };
+
```
Comment (ambroisie) [inline pkgs/misc/tmux-plugins/default.nix:547]: Let's keep the same formatting as the rest of the file

```suggestion
```

File: pkgs/misc/tmux-plugins/default.nix [hunk 2]
```diff
@@ -534,6 +534,24 @@ in rec {
     };
   };
 
+  rose-pine = mkTmuxPlugin {
+    pluginName = "rose-pine";
+    version = "unstable-2024-01-08";
+    rtpFilePath = "rose-pine.tmux";
```
Comment (ambroisie) [inline pkgs/misc/tmux-plugins/default.nix:540]: Is it necessary?

**Review by ambroisie — CHANGES_REQUESTED** [review]

File: pkgs/misc/tmux-plugins/default.nix (hunk 1 above)
Comment (AxelTLarsson) [reply pkgs/misc/tmux-plugins/default.nix:547]: Sorry will fix!

File: pkgs/misc/tmux-plugins/default.nix (hunk 2 above)
Comment (AxelTLarsson) [reply pkgs/misc/tmux-plugins/default.nix:540]: Yes, otherwise we get `rose_pine.tmux` which doesn't exist.

**Review by ambroisie — APPROVED** [review]

**Review by Janik-Haag — APPROVED** [review]

**Pull request merged**